Products/CMFPlone/resources/browser/styles.py

```diff
@@ -143,6 +152,18 @@ def styles(self):
                     'bundle': 'diazo'}
 
             result.append(data)
+
+        # custom.css
+        custom_css = {
+            'rel': 'stylesheet',
+            'conditionalcomment': '',
+            'src': "{0}/custom.css?timestamp={1}".format(
+                self.site_url,
+                self.custom_css_timestamp,
+            ),
+            'bundle': 'custom-css'
+        }
+        result.append(custom_css)
```

Comment:
Maybe it is better to add some condition to not always append the custom css?
It makes no sense if it is empty, right?